assignment_w48.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <cstdlib>
+#include <algorithm>
+#include <fstream>
+#include <vector>
+#include <string>
```

Comment:
avoid including unnecessary header files for better performance and faster compilation